iothub/device/src/ConnectionStatus.cs

```diff
@@ -4,32 +4,36 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    /// Connection status supported by the client.
+    /// The connection status from a connection change event.
     /// </summary>
     public enum ConnectionStatus
     {
         /// <summary>
         /// The device or module is disconnected.
-        /// <para>Inspect the associated <see cref="ConnectionStatusChangeReason"/> returned (and exception thrown, if any), and take appropriate action.</para>
         /// </summary>
+        /// <remarks>
+        /// Observe the associated <see cref="RecommendedAction"/> or <see cref="ConnectionStatusChangeReason"/> (and any exception thrown), and take appropriate action.
+        /// </remarks>
         Disconnected,
 
         /// <summary>
         /// The device or module is connected.
-        /// <para>The client is connected, and ready to be used.</para>
         /// </summary>
+        /// <remarks>The client is connected, and ready to be used.</remarks>
         Connected,
 
         /// <summary>
-        /// The device or module is attempting to reconnect.
-        /// <para>The client is attempting to recover the connection. Do NOT close or open the client instance when it is retrying.</para>
+        /// The client is attempting to reconnect per retry policy.
         /// </summary>
+        /// <remarks>The client is attempting to recover the connection. Do NOT close or open the client instance when it is retrying.</remarks>
         DisconnectedRetrying,
 
         /// <summary>
-        /// The device connection was closed.
-        /// <para>If you want to perform more operations on the device client, you should <see cref="IotHubDeviceClient.Dispose()"/> and then re-initialize the client.</para>
+        /// The device connection was closed gracefully.
         /// </summary>
-        Disabled,
+        /// <remarks>
+        /// To perform more operations on the client, call <see cref="IotHubDeviceClient.Dispose()"/> and then re-initialize the client.
+        /// </remarks>
+        Closed,
```

Comment:
Make sure you run these tests and make any changes as necessary: https://github.com/Azure/azure-iot-sdk-csharp/blob/main/e2e/test/iothub/ConnectionStatusChangeHandlerTests.cs